src/myq-garagedoor.ts

```diff
@@ -272,23 +244,14 @@ export class myQGarageDoor extends myQAccessory {
         void this.doorCommand(hap.Characteristic.TargetDoorState.OPEN);
       }
 
-      if(callback) {
-        callback(null);
-      }
-
       return true;
-
     }
 
     // HomeKit has told us something that we don't know how to handle.
     this.log.error("%s: Unknown SET event received: %s.", accessory.displayName, value);
 
-    if(callback) {
-      callback(new Error("Unknown SET event received: " + value.toString()));
-    }
-
+    //new Error("Unknown SET event received: " + value.toString()));
```

Comment:
not sure how you want to handle this?

Reply:
Stick to logging the error and let it keep running. This code path should never execute anyway...I've never seen it happen in all these years. :smile: